tests/integration/network_test.py

```diff
@@ -7,7 +7,6 @@
 from ..base import requires_api_version
 
 
-@requires_api_version('1.21')
```

Comment:
How about leaving this here, and only adding `requires('1.22')` to the few methods where it's necessary?